contrib/gel/gevd/gevd_noise.h

```diff
@@ -60,7 +60,7 @@ class gevd_noise
   static bool RealZeroCrossing(const float* dhist, const int nbin,
                                float& index);
  private:
-  gevd_noise() {} // no default constructor!
+  gevd_noise() = default; // no default constructor!
```

Comment:
This might be confusing to someone reading this later on 😄

Reply:
Fixed:  using =delete to meet comment specification, and the code still passes all tests.

Reply:
Great!